### PR TITLE
Adding a 400ZR test for ZR supply voltage.

### PR DESCRIPTION
**Pull request by ahsaanyousaf**
Adding a 400ZR test for ZR supply voltage.

### CODE REVIEW

File: feature/platform/transceiver/zr_supply_voltage_test/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# TRANSCEIVER-12: Telemetry: 400ZR Transceiver Supply Voltage streaming.
+
+## Summary
+
+Validate 400ZR transceivers report module level internally measured input supply
+voltage in 100 µV increments as defined in the CMIS.
+
+Link to CMIS:
+https://www.oiforum.com/wp-content/uploads/CMIS5p0_Third_Party_Spec.pdf
+
```
Comment (dplore) [inline feature/platform/transceiver/zr_supply_voltage_test/README.md:10]: Add test topology header

File: feature/platform/transceiver/zr_supply_voltage_test/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        example -10 dBm.
+      * Both transceivers are tuned to a valid centre frequency
+        example 193.1 THz.
+*   With the ZR link established as explained above, verify that the
+    following ZR transceiver telemetry paths exist and are streamed for both
+    the ZR optics.
+    *   /platform/components/component/state/supply-voltage/instant
+    *   /platform/components/component/state/supply-voltage/min
+    *   /platform/components/component/state/supply-voltage/max
+    *   /platform/components/component/state/supply-voltage/avg
```
Comment (dplore) [inline feature/platform/transceiver/zr_supply_voltage_test/README.md:27]: /components/component/transceiver/state/supply-voltage	

```suggestion
    *   /components/component/transceiver/state/supply-voltage/instant
    *   /components/component/transceiver/state/supply-voltage/min
    *   /components/component/transceiver/state/supply-voltage/max
    *   /components/component/transceiver/state/supply-voltage/avg
```

File: feature/platform/transceiver/zr_supply_voltage_test/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Config Parameter coverage
+
+*   /interfaces/interface/config/enabled
+
+## Telemetry Parameter coverage
+
+*   /platform/components/component/state/supply-voltage/instant
+*   /platform/components/component/state/supply-voltage/min
+*   /platform/components/component/state/supply-voltage/max
+*   /platform/components/component/state/supply-voltage/avg
```
Comment (dplore) [inline feature/platform/transceiver/zr_supply_voltage_test/README.md:62]:
```suggestion
    *   /components/component/transceiver/state/supply-voltage/instant
    *   /components/component/transceiver/state/supply-voltage/min
    *   /components/component/transceiver/state/supply-voltage/max
    *   /components/component/transceiver/state/supply-voltage/avg
```

**Review by dplore — CHANGES_REQUESTED** [review]

File: feature/platform/transceiver/zr_supply_voltage_test/README.md (hunk 1 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_supply_voltage_test/README.md:10]: Done

**Review by dplore — APPROVED** [review]

**Pull request merged**